content/content.md

```diff
@@ -390,58 +385,76 @@ As we add more and more of these images, we end up with something that becomes c
+- 您如何在數學上表示傅立葉轉換？
+- 連續時間傅立葉轉換和離散時間傅立葉轉換之間有什麼區別？
+- 如何計算傅立葉轉換？
+- 如何對整首歌曲進行傅立葉轉換？ （而不只是一個音符。）
 
-## Questions for the curious
+## 延伸問題
 
-I skipped most of the math stuff here, but if you're interested in the underlying principles of how it works, here are some questions you can use to guide your research:
+小波轉換(或稱小波分析, wavelet analysis)，就是由傅立葉轉換的基礎而衍伸出來的方法。而小波轉換的不同處在於，將原本無限長度的正弦波，轉換成了有限長度，並且會衰減的小載波。
 
-- How do you mathematically represent a Fourier transform?
-- What's the difference between a continuous time Fourier transform and a discrete time Fourier transform?
-- How do you computationally do a Fourier transform?
-- How do you do a Fourier transform of a whole song? (Rather than just a single note.)
+簡單來說，做傅立葉轉換能夠得到頻率，而做小波轉換，能夠得到時間和頻率。
 
-## Further 'reading'
+目前離散小波轉換（DWT）常被用於訊號編碼，像是工程和計算機科學，而連續小波轉換（CWT）常被用於訊號分析，即科學研究類。尤其在生醫領域中，心電圖 (Electrocardiography) 與腦波圖 (Electroencephalography) 是兩項常見的生醫應用。
 
-To learn more, some really good resources you can check out are:
+## 延伸閱讀
+
+要瞭解到更多的訊息，這裡有一些非常好用的資源可以參考(作者推薦的資源是英文版的)：
 
 [An Interactive Guide To The Fourier Transform](https://betterexplained.com/articles/an-interactive-guide-to-the-fourier-transform/)
-A great article that digs more into the mathematics of what happens.
+從數學角度更加深刻的介紹傅立葉轉換。
 
 [But what is the Fourier Transform? A visual introduction.](https://www.youtube.com/watch?v=spUNpyF58BY)
-A great Youtube video by 3Blue1Brown, also explaining the maths of Fourier transforms from an audio perspective.
+Blue1Brown 制作的 YouTube 影片，從音頻的角度解釋傅立葉轉換的數學原理。
 
 [A Tale of Math & Art: Creating the Fourier Series Harmonic Circles Visualization](https://alex.miller.im/posts/fourier-series-spinning-circles-visualization/)
-Another article explaining how you can use epicycles to draw a path, explained from a linear algebra perspective.
+從線性代數的角度解釋如何用周轉園來畫出形狀
 
 [Fourier transform (Wikipedia)](https://en.wikipedia.org/wiki/Fourier_transform)
-And of course, the Wikipedia article is pretty good too.
+當然，為基本科的解釋也很不錯的！
+
+以下是譯者為您推薦的一些中文參考資料：
+
+[傅立葉轉換 (維基百科 中文版)](https://zh.wikipedia.org/wiki/%E5%82%85%E9%87%8C%E5%8F%B6%E5%8F%98%E6%8D%A2)
 
-## The author
+[圖解傅立葉轉換](https://hackmd.io/@sysprog/fourier-transform)
+
+## 作者
 
 <canvas id="its-meee" class="sketch" width=500 height=500></canvas>
 
-I'm Jez! Full time I work at a [search company](https://www.google.com/) in the Bay Area, and in my spare time I like making games and interactive code things like this!
+我叫Jez! 我全職在美國海灣地區的[搜尋引擎公司](https://www.google.com/) 工作，在業餘時間我喜歡製作遊戲和互相交流程式碼。
+
+這個網頁是開源的，您可以在[GitHub](https://github.com/Jezzamonn/fourier)上查看程式碼! 如果您有任何回饋意見或想提出任何問題，請隨時透過email<span id="email-text">fourier [at] jezzamon [dot] com</span>發送電子郵件給我，或在tweetr上[Twitter](https://twitter.com/jezzamonn)發一個推文給我。
```

Comment:
When this is converted to HTML, it will look like:

這個網頁是開源的，您可以在GitHub上查看程式碼! 如果您有任何回饋意見或想提出任何問題，請隨時透過emailfourier@jezz<....>發送電子郵件給我

(I edited the end of the email to avoid spam bots 🙂)

Should there be a space between email and fourier?